### PR TITLE
Added the ability to output a visual representation of a checklist

### PR DESCRIPTION
**Pull request by njakobsen**
The image shows commits grouped by subject, and the connections between those groups.

### CODE REVIEW

**Comment by njakobsen** [comment]
Hmm. I think I still need to be able to visualize the connections created by `associated_records`.

File: lib/stagehand/auditor/checklist_visualizer.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            key = edge_key(entry_a, entry_b)
+            next if edges.include?(key)
+            next if @nodes[entry_a] == @nodes[entry_b]
+
+            edges << key
+            unless same_subject?(entry_a, entry_b)
+              create_edge(entry_a, entry_b, :color => :red, :fontcolor => :red, :label => edge_label(entry_a, entry_b))
+            end
+          end
+        end
```
Comment (rywall) [inline lib/stagehand/auditor/checklist_visualizer.rb]: Can we split this code into a couple methods? It's a bit hard to read all in one method.

Comment (njakobsen) [inline lib/stagehand/auditor/checklist_visualizer.rb]: I've refactored this as part of other changes. Let me know if it's clearer now.

**Comment by rywall** [comment]
Looks good to me.

**Pull request merged**